core/identity-hub/src/main/java/org/eclipse/edc/identityhub/IdentityHubExtension.java

```diff
@@ -46,10 +48,15 @@ public class IdentityHubExtension implements ServiceExtension {
     @Inject
     private TransactionContext transactionContext;
 
-    @Inject
     private CredentialEnvelopeTransformerRegistry credentialEnvelopeTransformerRegistry;
 
 
+    @Override
```

Comment:
Why not just:
```java
@Provider
public CredentialEnvelopeTransformerRegistry credentialEnvelopeTransformerRegistry() {
    return new CredentialEnvelopeTransformerRegistryImpl();
}
```

Reply:
I guess because the service it's also needed in `CollectionsWriteProcessor` which is in the default  impl of 
`MessageProcessorRegistry` service